OpenOversight/tests/routes/test_officer_and_department.py

```diff
@@ -1166,7 +1166,7 @@ def test_expected_dept_appears_in_submission_dept_selection(mockdata, client, se
 
 def test_admin_can_add_new_officer(mockdata, client, session, department, faker):
     with current_app.test_request_context():
-        login_admin(client)
+        rv, admin = login_admin(client)
```

Comment:
It doesn't look like we're using the rv, change that to an underscore, por favor.